packages/wrangler/src/metrics/send-event.ts

```diff
@@ -120,3 +120,66 @@ export async function sendMetricsEvent(
 		logger.debug("Error sending metrics event", err);
 	}
 }
+
+export type CommonEventProperties = {
+	/** The version of the wrangler client that is sending the event. */
+	wranglerVersion: string;
+	/**
+	 * The platform that the wrangler client is running on.
+	 */
+	osPlatform: string;
+	osVersion: string;
+	/**
+	 * The package manager that the wrangler client is using.
+	 */
+	packageManager: string | undefined;
+	nodeVersion: string;
+	/**
+	 * Whether this is the first time the user has used the wrangler client.
+	 */
+	isFirstUsage: boolean;
+
+	amplitude_session_id: number;
+	amplitude_event_id: number;
+	isCI: boolean;
+	isInteractive: boolean;
+	/**
+	 * A list of normalised argument names/flags that were passed in or are set by default.
+	 * Excludes boolean flags set to false.
+	 */
+	argsUsed: string[];
+	/**
+	 * Same as argsUsed except concatenated for convenience in Amplitude
+	 */
+	argsCombination: string;
```

Comment:
We will likely referring people to check this file if they wanna know more about what we are collecting. Shall we do some clean-up on this file? e.g. adding more comments on each property

Reply:
781b5f38979ac5ef8ed00de9c47a5fbe75cacde6
This file also contains some stuff about the old metrics implementation, which I guess will also be on by default now. Should we